code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -98,6 +98,14 @@
 	if(emagged == 2) //Emag functions
 		if(istype(loc,/turf/simulated))
 
+			for(var/mob/living/carbon/victim in src.loc)
+				if(victim.stat != DEAD)//cleanbots always finish the job
+					victim.visible_message("<span class='danger'>[src] sprays hydrofluoric acid at [victim]!</span>", "<span class='danger'>[src] sprays you with hydrofluoric acid!</span>")
+					victim.adjustFireLoss(25)

```

Comment:
Use an actual acid touch reaction if you want that. Change the text, and make sure it uses normal attack code so as to respect protections (armor, shields), if you are going with dealing burn damage only.